scripts/ramdom_seed_register.cpp

```diff
@@ -0,0 +1,97 @@
+#include <iostream>
+#include <fstream>
+#include <vector>
+#include <random>
+#include <algorithm>
+
+// Function to generate a random seed
+int generateSeed()
+{
+    std::random_device rd;
+    std::mt19937 gen(rd());
+    std::uniform_int_distribution<int> dist(1, 100000000);
+    return dist(gen);
+}
+
+// Function to check if a seed is in the used seeds file
+bool isSeedUsed(int seed, const std::string &filename)
+{
+    std::ifstream file(filename);
+    if (!file)
+    {
+        std::cerr << "Error opening file: " << filename << std::endl;
+        return false;
+    }
+
+    int usedSeed;
+    while (file >> usedSeed)
+    {
+        if (usedSeed == seed)
+        {
+            return true;
+        }
+    }
+
+    return false;
+}
+
+void writeRamdomSeedToFile(const std::vector<int> &generateSeedList, const std::string &filename)
```

Comment:
typo Ramdon => Random, same else where and in the filename.

Reply:
Oh my bad. I will fix and push it to the PR

Reply:
I prefer cpp. I was not sure if root works with cpp file extension. I will change it and see how it goes.